lib/Button/index.js

```diff
@@ -32,7 +34,8 @@ class Button extends Component {
     disableOnClick: PropTypes.bool,
     className: PropTypes.string,
     title: PropTypes.string,
-    isSubmit: PropTypes.bool
+    isSubmit: PropTypes.bool,
+    medium: PropTypes.bool
```

Comment:
instead of as a dedicated prop, why not just add it as a "type" like the rest of the other button styles?

Reply:
+1